test/helpers/index.coffee

```diff
@@ -0,0 +1,81 @@
+_ = require 'underscore'
+expect = chai.expect
+React = require 'react'
+ReactAddons    = require('react/addons')
+ReactTestUtils = React.addons.TestUtils
+{Promise}      = require 'es6-promise'
+{commonActions} = require './utilities'
+sandbox = null
+ROUTER = null
+CURRENT_ROUTER_PARAMS = null
+CURRENT_ROUTER_PATH   = null
+CURRENT_ROUTER_QUERY = null
+# Mock a router for the context
+beforeEach ->
+  sandbox = sinon.sandbox.create()
+  ROUTER  = sandbox.spy()
+  ROUTER.makeHref = sandbox.spy()
+  ROUTER.isActive = sandbox.spy()
+  ROUTER.transitionTo = sandbox.spy()
+  ROUTER.getCurrentPath = sandbox.spy( -> CURRENT_ROUTER_PATH )
+  ROUTER.getCurrentQuery = sandbox.spy( -> CURRENT_ROUTER_QUERY )
+  ROUTER.getLocation = sandbox.spy( ->
+    addChangeListener: sandbox.spy()
+    removeChangeListener: sandbox.spy()
+  )
+  ROUTER.match = sandbox.spy()
+  ROUTER.getCurrentParams = sandbox.spy( -> CURRENT_ROUTER_PARAMS )
```

Comment:
Yes, I think it makes sense to remove all router related stuff from here and leave that in a separate file in Tutor for now.  If we start using the router outside Tutor then the helpers could move then.  I'd neglected to do so right now because I'd basically just copied the file over :)  

I'll clean it up, thanks for pointing that out.